eng/Versions.props

```diff
@@ -1,10 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0">
   <PropertyGroup>
-    <VersionPrefix>1.0.0</VersionPrefix>
-    <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
+    <!-- We need to move to 4.7 as part of our versioning change when we switch to arcade to avoid downgrading versions -->
+    <MajorVersion>4</MajorVersion>
+    <MinorVersion>7</MinorVersion>
+
+    <!-- Always use shipping version insted of dummy version -->
+    <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
+    <PreReleaseVersionLabel>preview1</PreReleaseVersionLabel>
+    <!-- Opt-out repo features -->
+    <UsingToolXliff>false</UsingToolXliff>
+
+    <!-- Always use shipping version insted of dummy version -->
+    <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
```

Comment:
looks like copy paste overlook.

Reply:
Not sure I follow this is an intentional change.

Reply:
see 7 lines above, you duplicated the property.

Reply:
Ah yes a merge issue I'll clean that up.